docker/utbot_docker_dev.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
https://stackoverflow.com/questions/44662044/read-i-invalid-option-in-shell-on-osx
https://itnext.io/upgrading-bash-on-macos-7138bd1066ba